backend/db/manage_db.py

```diff
@@ -1 +1,83 @@
-# TODO create the database for the app
+from flask import Flask
+from sqlalchemy import create_engine  
+from sqlalchemy import Column, String, Float, Boolean
+from sqlalchemy.dialects.postgresql import UUID
+from sqlalchemy.ext.declarative import declarative_base  
+from sqlalchemy.orm import sessionmaker
+
+from flask_script import Manager
+from flask_migrate import Migrate, MigrateCommand
+
+from flask_sqlalchemy import SQLAlchemy
+
+import psycopg2
```

Comment:
Looks like we're not using `psycopg2` anymore, so I think we can remove this import